specs/_features/eip7805/p2p-interface.md

```diff
@@ -0,0 +1,87 @@
+# EIP-7805 -- Networking
+
+This document contains the consensus-layer networking specification for EIP-7805.
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Time parameters](#time-parameters)
+- [Configuration](#configuration)
+- [The gossip domain: gossipsub](#the-gossip-domain-gossipsub)
+  - [Topics and messages](#topics-and-messages)
+    - [Global topics](#global-topics)
+      - [`inclusion_list`](#inclusion_list)
+- [The Req/Resp domain](#the-reqresp-domain)
+  - [Messages](#messages)
+    - [InclusionListByCommitteeIndices v1](#inclusionlistbycommitteeindices-v1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+### Time parameters
+
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `attestation_deadline` | `uint64(4)` | seconds | 4 seconds |
+
+### Configuration
+
+| `MAX_REQUEST_INCLUSION_LIST` | `2**4` (= 16) | Maximum number of inclusion list in a single request |
+
+### The gossip domain: gossipsub
+
+#### Topics and messages
+
+The new topics along with the type of the `data` field of a gossipsub message are given in this table:
+
+| Name                          | Message Type                                         |
+|-------------------------------|------------------------------------------------------|
+| `inclusion_list`    | `SignedInclusionList` [New in EIP-7805] |
+
+##### Global topics
+
+EIP-7805 introduces a new global topic for inclusion lists.
+
+###### `inclusion_list`
+
+This topic is used to propagate signed inclusion list as `SignedInclusionList`.
+The following validations MUST pass before forwarding the `inclusion_list` on the network, assuming the alias `message = signed_inclusion_list.message`:
+
```

Comment:
Aren't we also limiting IL size by `MAX_BYTES_PER_INCLUSION_LIST = 8192` bytes as indicated in the EIP?